lib/router/mbaas-spec.js

```diff
@@ -0,0 +1,46 @@
+var _ = require('lodash');
+const assert = require('assert');
+const authHandler = require('./mbaas');
+const sampleUserConfig = require('../../test/fixtures/sampleUserConfig.json');
+const sampleProfileData = require('../../test/fixtures/sampleUserProfile.json');
+const sampleProfileDataLength  = Object.keys(sampleProfileData).length;
+
+var sampleExclusionList1 = ['banner'];
+var sampleExclusionList2 = ['banner', 'avatar'];
+var sampleExclusionList3 = [];
```

Comment:
What happens if `authResponseExclusionList` is `null` or `undefined` ? Would be good to test that.